Mathlib/LinearAlgebra/Finsupp/LinearCombination.lean

```diff
@@ -223,7 +223,6 @@ theorem span_image_eq_map_linearCombination (s : Set α) :
       by_cases h : c ∈ s
       · exact smul_mem _ _ (subset_span (Set.mem_image_of_mem _ h))
       · simp [(Finsupp.mem_supported' R _).1 hz _ h]
-    -- Porting note: `rw` is required to infer metavariables in `sum_mem`.
     rw [mem_comap, linearCombination_apply]
```

Comment:
Does this still apply? Is it worth mentioning?

Reply:
I'm confused how one would expect this to work without the `rw`, so I think it's fine to remove the porting note.